.env.example

```diff
@@ -2,11 +2,11 @@ AP_ENGINE_EXECUTABLE_PATH=dist/packages/engine/main.js
 AP_FRONTEND_URL=http://localhost:8080
 AP_NODE_EXECUTABLE_PATH=/usr/local/bin/node
 AP_POSTGRES_DATABASE=activepieces
-AP_POSTGRES_HOST=postgres
-AP_POSTGRES_PASSWORD=A79Vm5D4p2VQHOp2gd5
-AP_POSTGRES_PORT=5432
-AP_POSTGRES_USERNAME=postgres
 AP_REDIS_HOST=redis
 AP_REDIS_PORT=6379
 AP_TELEMETRY_ENABLED=true
 AP_SANDBOX_RUN_TIME_SECONDS=600
+AP_POSTGRES_HOST=postgres
+AP_POSTGRES_PORT=5432
+AP_POSTGRES_USERNAME=postgres
+## AP_POSTGRES_PASSWORD=CHANGE_ME
```

Comment:
The example file is not used so no need to comment out a line.
Also please note that I always keep the variable list sorted alphabetically for easier access and you always miss the sort 😭